src/style.css

```diff
@@ -2,2 +1,73 @@
+body,
+ul,
+ol,
+li {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
   margin: 0;
+  list-style: none;
+}
+
+.MOYO {
+  display: block;
+  cursor: pointer;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+.head {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px #0000000d;
+}
+
+.list {
+  display: flex;
+  margin: 0;
+}
+
+.link {
+  margin-right: 20px;
+}
+
+.link:last-child {
+  margin-right: 0;
+}
+
+.item {
+  color: black;
```

Comment:
Ensure sufficient color contrast between the text and background for better readability and accessibility. The current color might need to be checked against the background color.